pkgs/applications/audio/rhythmbox/default.nix

```diff
@@ -83,6 +87,13 @@ in stdenv.mkDerivation rec {
     "--with-libsecret"
   ];
 
+  preFixup = ''
+    gappsWrapperArgs+=(
+      --prefix PYTHONPATH : "${python3.pkgs.pygobject3}/lib/${python3.libPrefix}/site-packages/"
```

Comment:
You can use `${python3.sitePackages}` instead of `lib/${python3.libPrefix}/site-packages`

Reply:
Okay